litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/__tests__/ChooseWorkflow.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render } from '../../../../testHelpers/test-util';
+import ChooseWorkflow from '../index';
+import '@testing-library/jest-dom';
+
+describe('ChooseWorkflow', () => {
+  it('Renders header, <RadioGroup>, <Accordion', () => {
+    const { getByTestId } = render(<ChooseWorkflow />);
+    expect(getByTestId('chooseworkflow-header')).toBeTruthy();
```

Comment:
try to write the test case with actual states to check the logic

Reply:
Sure, will try to incorporate.